### ISSUE TITLE
A11y_AzureDataStudio_Notebooks-Create_Keyboard: Not able to access the toolbar using keyboard keys.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#22676]
"[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### #GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #SH-AzureDataStudio-Linux-Jul2021; #Azure Data Studio; #WCAG2.1.1; #Linux; #Keyboard; #Closed; #Regressed:05-15-23

### Environment Details:​
Application Name: MySQL extension for Azure Data Studio
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

### Repro Steps:

1. Launch Azure Data Studio application.
2. TAB to "File" and hit ENTER key. TAB to "New Notebook" and hit ENTER key.
3. TAB to "Cell" dropdown and hit ENTER. TAB to "Cell Code" / "Cell Text" and hit ENTER key.
4. Navigate to the toolbar using keyboard keys and observe that not able to access the toolbar using keyboard keys.

### Actual Result:​
Not able to access the toolbar using keyboard keys.

### Expected Result:​
Toolbar should be accessible using keyboard keys.

### User Impact:
Keyboard users will not be able to access the toolbar using keyboard and will not be able to use the different toolbars.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Attachment:

[Github issue for keyboard keys.webm](https://user-images.githubusercontent.com/90901519/228770370-9fd278e5-5eb1-4895-b9b5-7a80d2841724.webm)

**Comment by chlafreniere** [comment]
@PoojaNamde, pressing the Escape key will get the user out of edit mode for a notebook cell. Pressing Tab/Shift+Tab should allow the user to access the cell level toolbar.

If you press Escape first, can you confirm that you are able to access the cell-level toolbar?

**Comment by Swati1700** [comment]
@chlafreniere After pressing "Escape" and "Shift/Tab" key we are unable to access the cell-level toolbar.

**Comment by alanrenmsft** [comment]
@Swati1700 @PoojaNamde I was able to access the toolbar, please see the screen recording below.

![keyboard-nav](https://github.com/microsoft/azuredatastudio/assets/13777222/6488d9b9-e3a8-4b4e-87bc-ad50e8b83882)

**Comment by PoojaNamde** [comment]
Verified this issue in the latest environment
Version: 1.45.0-insider (user setup)
Commit: 4329d04da3b86689556d4dceab7d9b3d5cde3f58
Date: 2023-05-15T05:30:26.887Z
VS Code: 1.70.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
 
issue is fixed able to access toolbar 1st using ESC key and after by Shift + TAB key. hence closing the bug.
